src/other/contango/ContangoWitch.sol

```diff
@@ -180,59 +183,45 @@ contract ContangoWitch is Witch, IContangoWitch {
             uint256 topUpAmount = requiredArtIn - artIn;
```

Comment:
I think we should transform this into `debtToBase` before calling the insurance fund